src/wazuh.py

```diff
@@ -73,3 +79,51 @@ def install_certificates(container: ops.Container, public_key: str, private_key:
     """
     container.push(CERTIFICATES_PATH / "filebeat.pem", public_key, make_dirs=True)
     container.push(CERTIFICATES_PATH / "filebeat-key.pem", private_key, make_dirs=True)
+
+
+def configure_git(container: ops.Container, git_repository: str, git_ssh_key: str) -> None:
+    """Configure git.
+
+    Args:
+        container: the container to configure git for.
+        git_repository: the git repository to add to known hosts.
+        git_ssh_key: the SSH key for the git repository.
+    """
+    hostname = git_repository.split("@")[1].split("/")[0]
+    process = container.exec(["ssh-keyscan", "-t", "rsa", hostname])
```

Comment:
Does this mean we're depending on the type of ssh key being "rsa"? If so are we documenting that or checking that somewhere?

Reply:
We need to add the hostname to the .known_hosts file and that's the way :)

Reply:
You're running `ssh-keyscan` with "type == rsa", so isn't that going to fail if someone specifies a key of a different type (e.g. `ed25519`)?

Reply:
This doesn't validate the user's key but the server's key. It's equivalent to confirming "yes" when opening an SSH connection for the first time

Reply:
I see, thx.